guard.cpp

```diff
@@ -7,43 +7,89 @@
 using namespace std;
 using namespace openpower::guard;
 
-void guardList()
+void printHeader(bool display)
+{
+    if (display)
+    {
+        std::cout << "ID       | ERROR    |  Type  | Path " << std::endl;
+    }
+    else
+    {
+        std::cout << "No Records to display" << std::endl;
+    }
+}
+
+void guardList(bool displayResolved)
 {
     auto records = getAll();
     if (!records.size())
     {
-        std::cout << "No Records to display" << std::endl;
+        std::cout << "No "
+                  << (displayResolved == true ? "resolved" : "unresolved")
```

Comment:
Do we really need to mention `resolved` vs `unresolved` here? We don't have any records right i.e guard file is empty?